packages/components/psammead-script-link/src/index.test.jsx

```diff
@@ -17,18 +17,23 @@ describe('ScriptLink', () => {
   );
 
   it('should render a link', () => {
+    const onClickMock = jest.fn();
+
     const { container } = render(
-      <ScriptLink {...props} variant="lat">
+      <ScriptLink {...props} variant="lat" onClick={onClickMock}>
         Lat
       </ScriptLink>,
     );
+
     const links = container.querySelectorAll('a');
+    fireEvent.click(links[0]);
 
     expect(links).toHaveLength(1);
     expect(links[0]).toHaveProperty('href');
     expect(links[0].href).toEqual('https://www.bbc.co.uk/news');
     expect(links[0].textContent).toEqual('Lat');
     expect(links[0].dataset).toHaveProperty('variant');
     expect(links[0].dataset.variant).toEqual('lat');
+    expect(onClickMock).toHaveBeenCalled();
```

Comment:
It's worth considering splitting these tests out https://github.com/ryanmcdermott/clean-code-javascript#testing

Reply:
Thanks for the link. Changes applied.